packages/angular_devkit/build_angular/src/builders/browser-esbuild/options.ts

```diff
@@ -203,14 +203,19 @@ export async function normalizeOptions(
   };
 }
 
+/** The list of valid config files can be found https://github.com/tailwindlabs/tailwindcss/blob/ba5454543e74a6d702ce11b410d27672c2ee4b3f/src/util/resolveConfigPath.js#L4-L9*/
+const tailwindConfigFiles = [
+  './tailwind.config.js',
+  './tailwind.config.cjs',
+  './tailwind.config.mjs',
+  './tailwind.config.ts',
+];
```

Comment:
With the increased config file count, this causes a worst case of 8 file exist checks.
Performance-wise, it may be better to perform two async readdir calls in parallel and then check the entry lists for the relevant files.

Reply:
Good call.